src/tests/unit/tests/DetailsView/components/assessment-instance-table.test.tsx

```diff
@@ -150,46 +162,54 @@ describe('AssessmentInstanceTable', () => {
                 instanceCount: 1,
             } as DefaultMessageInterface;
 
-            const testSubject = shallow(<AssessmentInstanceTable {...props} />);
-            expect(testSubject.getElement()).toEqual(defaultMessage.message);
+            render(<AssessmentInstanceTable {...props} />);
+            const hasMessage = screen.getByText('Message from getDefaultMessage');
+            expect(hasMessage).toBeDefined();
 
             getDefaultMessageMock.verifyAll();
         });
 
-        it("delegates the underlying list's onItemInvoked to the handler's updateFocusedTarget", () => {
+        it("delegates the underlying list's onItemInvoked to the handler's updateFocusedTarget", async () => {
             const fakeItem = { instance: { target: ['fake-instance-target-0'] } };
-            assessmentInstanceTableHandlerMock
-                .setup(a => a.updateFocusedTarget(fakeItem.instance.target))
-                .verifiable(Times.once());
 
-            const testSubject = mount(<AssessmentInstanceTable {...props} />);
-            testSubject.find(DetailsList).prop('onItemInvoked')(fakeItem);
+            useOriginalReactElements('@fluentui/react', ['Spinner', 'DetailsList', 'ActionButton']);
+            render(<AssessmentInstanceTable {...props} />);
+            const rowClick = screen.getAllByRole('row');
 
-            assessmentInstanceTableHandlerMock.verifyAll();
+            fireEvent.click(rowClick[1], fakeItem);
+            getMockComponentClassPropsForCall(DetailsList).onItemInvoked(fakeItem);
```

Comment:
Hi @madalynrose 
Cannot remove the fireEvent.click(rowClick[1], fakeItem);
because it is decreasing the test percentage coverage, attached the below screenshot for your reference.
![image](https://github.com/microsoft/accessibility-insights-web/assets/155614445/9a62bc25-ffe4-479a-9983-679c4c8eabfb)
Thank you